### PR TITLE
fix(Mint/Airdrop tokens): Darkmode images

### LINKED ISSUE #10299: [Community Mint / Airdrop Settings] Update mint / airdop image without background
### Description

Update images to be shown properly in dark mode:

Screenshot 2023-04-14 at 10 29 04

And:

Screenshot 2023-04-14 at 10 26 23

### PR DESCRIPTION
**Pull request by noeliaSD**
Closes #10299

### What does the PR do

Updated images without background.

### Screenshot of functionality 
Screenshot 2023-04-14 at 10 29 04

Screenshot 2023-04-14 at 10 28 59

Screenshot 2023-04-14 at 10 26 27

Screenshot 2023-04-14 at 10 26 23

### CODE REVIEW

**Comment by noeliaSD** [comment]
It just needs confirmation from @xAlisher that the `airdop` image is correct with the shadow.

**Comment by xAlisher** [comment]
> It just needs confirmation from @xAlisher that the `airdop` image is correct with the shadow.

If images with a shadow are bigger than others I suggest cutting the shadow.

**Comment by noeliaSD** [comment]
> > It just needs confirmation from @xAlisher that the `airdop` image is correct with the shadow.
> 
> If images with a shadow are bigger than others I suggest cutting the shadow.

Here with the image without shadow:
Screenshot 2023-04-17 at 08 59 51

File: ui/imports/assets/png/community/mint2_1.png
[the remark this answers is not in the document]
Comment (noeliaSD) [reply ui/imports/assets/png/community/mint2_1.png:1]: > Could be smaller, e.g. 512x512 like the other image

@xAlisher ??

[the remark this answers is not in the document]
Comment (caybro) [reply ui/imports/assets/png/community/mint2_1.png:1]: I mean, the image itself is fine but it's just too big for what we need if we want to display 256x256; just resize it down to 512

[the remark this answers is not in the document]
Comment (noeliaSD) [reply ui/imports/assets/png/community/mint2_1.png:1]: Will resize!!

[the remark this answers is not in the document]
Comment (noeliaSD) [reply ui/imports/assets/png/community/mint2_1.png:1]: Will resize!!

**Review by caybro — APPROVED** [review]

**Review by micieslak — APPROVED** [review]
LGTM!

**Pull request merged**